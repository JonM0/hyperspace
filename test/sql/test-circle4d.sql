select circle4d '((0,0,0,0), 1)';
select circle4d '((0,1,2,3), 2)';
select circle4d '((0,0,0,0), -0.8)';

select point4d '(0,0,0,0)' <@ circle4d '((0,0,0,0), 1)';
select point4d '(0,1,1,0)' <@ circle4d '((0,0,0,0), 1)';
select point4d '(0,0,1,0)' <@ circle4d '((0,0,0,0), 1)';

select point4d '(0.707,0.707,0,0)' <@ circle4d '((0,0,0,0), 1)';
select point4d '(0.708,0.708,0,0)' <@ circle4d '((0,0,0,0), 1)';

select circle4d('(0,0,0,0)', 1);
select circle4d('(0,0,0,0)', -23.9);