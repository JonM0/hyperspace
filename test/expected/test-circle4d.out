select circle4d '((0,0,0,0), 1)';
   circle4d    
---------------
 ((0,0,0,0),1)
(1 row)

select circle4d '((0,1,2,3), 2)';
   circle4d    
---------------
 ((0,1,2,3),2)
(1 row)

select circle4d '((0,0,0,0), -0.8)';
    circle4d     
-----------------
 ((0,0,0,0),0.8)
(1 row)

select point4d '(0,0,0,0)' <@ circle4d '((0,0,0,0), 1)';
 ?column? 
----------
 t
(1 row)

select point4d '(0,1,1,0)' <@ circle4d '((0,0,0,0), 1)';
 ?column? 
----------
 f
(1 row)

select point4d '(0,0,1,0)' <@ circle4d '((0,0,0,0), 1)';
 ?column? 
----------
 t
(1 row)

select point4d '(0.707,0.707,0,0)' <@ circle4d '((0,0,0,0), 1)';
 ?column? 
----------
 t
(1 row)

select point4d '(0.708,0.708,0,0)' <@ circle4d '((0,0,0,0), 1)';
 ?column? 
----------
 f
(1 row)

select circle4d('(0,0,0,0)', 1);
   circle4d    
---------------
 ((0,0,0,0),1)
(1 row)

select circle4d('(0,0,0,0)', -23.9);
     circle4d     
------------------
 ((0,0,0,0),23.9)
(1 row)

